beads_email/src/main/java/com/beads/email/service/EmailSenderService.java

```diff
@@ -21,17 +21,20 @@
 @Transactional
 public class EmailSenderService {
 
-    @Resource(name = OrderDaoImpl.BEAN_NAME)
     private OrderDao orderDao;
+    private EmailSender emailSender;
 
     @Autowired
-    private EmailSender emailSender;
+    public EmailSenderService(EmailSender emailSender,
+                              @Qualifier(value = OrderDaoImpl.BEAN_NAME) OrderDao orderDao) {
+        this.emailSender = emailSender;
```

Comment:
please add `requireNonNull(orderDao, "orderDao can't be null")`.
This will allow to be sure that objection will be instantiated correctly without and NE
it is good practice to follow